.github/workflows/prepare-release.yaml

```diff
@@ -0,0 +1,44 @@
+name: Prepare release PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to publish, e.g. 0.0.1'
+        required: true
+        default: '0.0.1'
+        type: string
+
+jobs:
+  prepare_release:
+    name: Prepare release PR
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: ./.github/actions/setup-and-build
+
+      - name: Bump versions
+        run: >
+          pnpm --recursive
+          --filter "@tutorialkit/*"
+          --filter create-tutorial
+          exec pnpm version --no-git-tag-version --allow-same-version ${{ inputs.version }}
+
+      - name: Generage changelog
+        run: |
+          pnpm run changelog
+          git checkout ./packages/cli
```

Comment:
I think that we should keep the CLI changelog here. 

IIRC once we set the tag, the changelog will use the latest tag known in git and use that to update the changelog. So that means that then it will generate an empty changelog for the CLI.

But I might be wrong.

Have you verified that doing it later works?

Reply:
Changelog for CLI is indeed empty after these steps. Let's keep the CLI's changelog here. 👍